docs/how-to/h-deploy-azure.md

```diff
@@ -0,0 +1,248 @@
+# How to deploy on Azure
+
+[Azure](https://azure.com/) is the cloud computing platform developed by Microsoft. It has management, access and development of applications and services to individuals, companies, and governments through its global infrastructure. Access the Azure web console at [portal.azure.com](https://portal.azure.com/).
+
+## Summary
+
+* [Install Azure and Juju tooling](#install-client-environment)
+  * [Authenticate](#authenticate)
+* [Bootstrap Juju controller on Azure](#bootstrap-juju-controller-on-azure)
+* [Deploy charms](#deploy-charms)
+* [Clean up](#clean-up)
+
+---
+
+## Install Client Environment
+
+> **WARNING**: The current limitations:
```

Comment:
It would be nice to implement proper admonitions (note blocks) in this PR instead of quotation. Not a blocker.